assets/src/styles/blocks/SplitTwoColumns.scss

```diff
@@ -79,7 +79,7 @@
   }
 
   @include large-and-up {
-    font-size: 4.125rem;
+    font-size: $font-size-xxl;
```

Comment:
Maybe you can just remove this entire `large-and-up` query? Since the font-size is the same as the one set in [just before](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/blob/37bb64fefc55b4ac2958d62e0566f7e3dc63f1f0/assets/src/styles/blocks/SplitTwoColumns.scss#L75) 🙂 
And actually you can also probably remove the one set [just after](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/blob/37bb64fefc55b4ac2958d62e0566f7e3dc63f1f0/assets/src/styles/blocks/SplitTwoColumns.scss#L86) since it's also the same value 🤔

Reply:
yes, indeed 👍  Thanks 😊